web/services/hooks/requests.tsx

```diff
@@ -46,7 +46,32 @@ const useGetRequest = (requestId: string) => {
             } as FilterNode,
           }),
         }
-      ).then((res) => res.json() as Promise<Result<HeliconeRequest, string>>);
+      );
+
+      const result = (await response.json()) as Result<HeliconeRequest, string>;
+
+      // update below logic to work for single request
+      const request = result.data;
+
+      if (request?.signed_body_url) {
+        try {
+          const contentResponse = await fetch(request.signed_body_url);
+          if (contentResponse.ok) {
+            const text = await contentResponse.text();
+
+            const content = JSON.parse(text) as {
+              request: string;
+              response: string;
+            };
+            request.request_body = content.request;
+            request.response_body = content.response;
+          }
+        } catch (error) {
+          console.log(`Error fetching content: ${error}`);
+        }
+      }
```

Comment:
TODO @ScottMktn we should Lazy load this in the frontend.

Reply:
in a future PR**

We don't want to wait for all the S3 calls to finish before returning the the frontend. We can return all the other fields for now and then lazy load the bodies in after the fact